concepticondata/conceptlists/Tolmie-1884-211.tsv

```diff
@@ -0,0 +1,212 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Tolmie-1884-211-1	1	Man	1554	MAN
+Tolmie-1884-211-2	2	Woman	962	WOMAN
+Tolmie-1884-211-3	3	Boy	1366	BOY
+Tolmie-1884-211-4	4	Girl	1646	GIRL
+Tolmie-1884-211-5	5	Infant	1289	BABY
+Tolmie-1884-211-6	6	My father (said by son)		
+Tolmie-1884-211-7	7	My father (said by daughter)		
+Tolmie-1884-211-8	8	My mother (said by son)		
+Tolmie-1884-211-9	9	My mother (said by daughter)		
+Tolmie-1884-211-10	10	My husband		
+Tolmie-1884-211-11	11	My wife		
+Tolmie-1884-211-12	12	My son (said by father)		
```

Comment:
kinship terms that depend on the EGO are in fact in part available in concepticon. I'd also say that the "my" construct here can be savely ignored. It is a phrase they use for elicitation, as this triggers the word, similar to using the imperative in verbal elicitation, and we tend to link those, unless there's an elicitation of the word for "father" or similar in isolation. So I'd link husband, wife, elder brother, elder sister, etc., but maybe leave the father-mother block unlinked, or add this as a distinction (there's "papa" vs, "father", and then there's the perspective of male egos, etc., maybe something we'll also find useful for kinbank and other projects?)